assets/css/desk-home.css

```diff
@@ -0,0 +1,396 @@
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  width: 100%;
+}
+
+.header-above {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  color: #fff;
+  background-color: #272a31;
+  gap: 14px;
+  width: 100%;
+  padding: 5px 10% 0;
+}
+
+.logo {
+  padding: 0 49px;
+}
+
+.header-above-content {
+  list-style-type: none;
+  font-size: 12px;
+}
+
+.header-below {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding: 0 10%;
+}
+
+.links {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.header-below-list {
+  display: flex;
+  align-items: center;
+
+  /* padding-right: 10%; */
+  justify-content: flex-end;
+  gap: 7%;
+}
+
+.header-below-content {
+  list-style-type: none;
+}
+
+.cc-btn {
+  border: 4px solid #ec5242;
+  padding: 12px;
+  white-space: nowrap;
+}
+
+.background-img {
+  background-image: url(../images/background-desktop/cross.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center bottom;
+  opacity: 0.2;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: -2;
+}
+
+.conference-page {
+  position: relative;
+  width: 100%;
+  padding: 0 10%;
+}
+
+.conference-content {
+  padding: 90px 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  z-index: 9;
+}
+
+.presents {
+  color: #ec5242;
+  font-size: 24px;
+  font-weight: 300;
+  line-height: 24px;
+}
+
+.conference-intro {
+  color: #ec5242;
+  font-size: 56px;
+  font-weight: 800;
+  font-family: "Lato", sans-serif;
+  line-height: 63px;
+}
+
+.first-text {
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 30px;
+  border-radius: 6px;
+  border: 4px solid #fff;
+  background-color: #d3d3d3;
+  padding: 12px;
+  color: #272a31;
+  text-align: center;
+  width: 100%;
+}
+
+.program-date {
+  font-size: 26px;
+  font-weight: 600;
+  line-height: 24px;
+}
+
+.location {
+  color: #665e5e;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 18px;
+  padding-bottom: 3rem;
+}
+
+.speakers-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0 10%;
+  gap: 3rem;
+}
```

Comment:
- [ ] Please ensure that all your speakers content and it detail matches the given design requirement, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student main speakers desktop](https://github.com/Kwamay/Capstone-Project/assets/43528594/19fb0f63-fd56-4164-be4a-a1457fd7b38b)

</td>
      <td>

![Design speakers desktop](https://github.com/Kwamay/Capstone-Project/assets/43528594/1e11a507-ef5c-4210-9ddf-3695ced36879)

</td>
    </tr>
  </table>